### ISSUE TITLE
Pull request

### ISSUE THREAD

**Issue by JeffLabonte** [rcasero/doc#15]
Hi,  

I have installed ubuntu 17.10 on my machine and I had some issues with my laptop. I found the solution to all my issues and I would like to share the solution to everyone through your doc. Would it be possible to enable me to make a pull request?

So I added the way to install bumblebee on a XPS 15 which would help a lot on the battery life!

Thank you for the taking the time to read me!

**Comment by rcasero** [comment]
Hi @JeffLabonte ,

Isn't bumblebee deprecated, and instead nvidia-prime is to be used? From this comment, e.g.

https://askubuntu.com/a/758995

> Bumblebee had no development since 2013 and is deprecated in 16.04, installation of nvidia-361 and nvidia-prime should take care of the bumblebee functionality. pilot6 gave the usage details here.

**Comment by JeffLabonte** [comment]
Yes @rcasero , you are right. Then I will add your comment to the doc and talk about how to use nvidia-prime instead. Would be good for you?

**Comment by rcasero** [comment]
That could be useful, yes. I notice a substantial difference in battery life between booting in Windows or linux. However, I think github doesn't support pull requests to wiki pages. As far as I know, the only way to add your changes would be that you upload your changes as a diff file here, and then I can review your suggested changes, and patch the document on a local clone of the wiki, and name you in the commit.

Alternatively, maybe it'd be better to have the documentation as an .md file in the project itself, rather than in the wiki. That'd allow pull requests.

**Comment by JeffLabonte** [comment]
Here is a gist with the diff I made on your file. [Here is the link](https://gist.github.com/JeffLabonte/1fccb3fb86fe1115053c13117ed5018d)

I have kept the bumblebee part but i added a deprecated tag. I added the PRIME section to explain how to use it and how to know that your system supports it.

I hope you like it!
